app/js/question_controller.js

```diff
@@ -166,7 +166,8 @@
     evt.preventDefault();
     var comment = question_field.value;
 
-    if (comment === '') {
+    // Disable submit if the comment is blank or with some spaces
+    if (comment === '' || comment.replace(/\s/g, '') === '') {
```

Comment:
You can also ditch the comment because `trim()` is enough to express the intent.